swap_meet/item.py

```diff
@@ -0,0 +1,47 @@
+#___________________WAVE 2_____________________________
+
+class Item:
+    def __init__(self, category = None, condition = None, age = None):
+        
+        if category is None:
+           self.category = ""
+        else:
+            self.category = category
+        
+        if condition is None:
+            self.condition = 0
+        else:
+            self.condition = condition
+        #_______________optional enhancement______________
+               
+        if age is None:
+            self.age = 0
+        else:
+            self.age = age
```

Comment:
The we can simplify the constructor body
```suggestion
           self.category = category
           self.condition = condition
           self.age = age
```